pre_marriage.html

```diff
@@ -10,17 +10,151 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         
-        <link href="images/mynikka.jpg" rel="shortcut icon">
-        <link rel="stylesheet" href="css/bootstrap.min.css"/>
-        <link rel="stylesheet" href="css/pre_married.css"/>
+        <link href="img/mynikka.jpg" rel="shortcut icon">
+        <link rel="stylesheet" href="css/bootstrap.min.css"/>  
+        <link rel="stylesheet" href="css/my-nikka.css"/>
         <script src="js/bootstrap.min.js"></script>
         <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
         <script src="js/my-nikka.js"></script>        
     </head>
     <body>
-        <nav class="collapse navbar-collapse" id="navbar"></nav>
+        <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+            <div class="container">
+                <div class="navbar-collapse collapse" id="navbar">
+                </div>
+            </div>
+        </nav>
+
+        <div class='container beneath-nav'>
+            <h1 style="margin-top: 70px;">Pre-Marriage</h1>
+            <p>("Here's a bunch of links for you to read regarding the approach of finding someone")</p>
+        </div>
         
-        <h1>Pre-Marriage</h1>
-        <h2>("Here's a bunch of links for you to read regarding the approach of finding someone")</h2>
+        <div class="container">
+            <h2>Phase 1 - <em>Don't even think about it until you're past these steps</em></h2>
+            <h3> 1.1 Prayer - <em>If you can’t commit a few minutes a day to God, then you’re not ready to commit to another person for a lifetime</em></h3>
+            <ul>
+                <li><h4><a class="links" id="sweetness_of_salah" href="https://www.youtube.com/playlist?list=PLm2Fyooe0tC23bv48VGfu51078jrfDU2D">Sweetness of Salah</a>
```

Comment:
I have an eye on maintainability here: instead of doing this, can we write a js widget that creates these for us? That way, all we need to do is feed it a json array and it'll create these tags for us. This is better for a few reasons:

1. Separates data and rendering
2. Allows data to come from other sources
3. Gives us flexibility. In case we want to add a backend, we'll just have to write a simple connection layer (or hook it up through a managed connection layered framework)
4. Makes maintenance of these pages easier
5. Keeps code size small